pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -808,7 +848,49 @@ log(404, "NOT_FOUND", "this", "page", "is", "not", "found");
 ```
 
 ## `[@bs.inline]`
-TODO
+
+Especially for library authors, `bs.inline` could be interesting. It tells the compiler explicitly to inline a constant directly. 
+
+Suppose you have the following binding for the node environment variable:
+
+```reason
+/* NodeEnv.re */
+[@bs.val] [@bs.scope "process.env"] external env: string = "NODE_ENV";
```

Comment:
I would love to have an example that doesn't require the knowledge of 2 other bs.decorators, even if the example isn't a real-world one.

For example:
```reason
[@bs.inline]
let color = "green"
```

Reply:
Do you mean 

```reason
[@bs.inline]
let green = "green"
```

in a Colors.re file? Would be even more fitting imho. And that's actually the second use-case besides the node env that I had for it, so good idea!

Reply:
Agreed, i think it should be enough to show the JS output `inline` vs `non-inline` to get the point across? After that, another example can explain that this is often used for injecting NODE_ENV for bundlers etc